src/components/Sidebar/index.ts

```diff
@@ -1,5 +1,5 @@
 export * from './Sidebar';
-export type { SidebarCTAProps } from './SidebarCTA';
 export type { SidebarCollapseProps } from './SidebarCollapse';
+export type { SidebarCTAProps } from './SidebarCTA';
```

Comment:
I'm confused if you have a custom prettier config somehow or have just made these changes of your own accord, but they fail `yarn prettier` so you need to revert them. I'm not sure why they were made anyway in a PR for the Accordion